### DISCUSSION TITLE
How to understand these parameters in GPS API?

### DISCUSSION THREAD

**Discussion by MJy1023** [microsoft/AirSim#4602] [Support Q&A]
enum GnssFixType : unsigned char

enum PositionCovarianceType : unsigned charenum

NavSatStatusServiceType : unsigned short int

**Comment by alonfaraj** [comment]
Hi @Jason-M1024,
The gps api follows ROS NavSatFix message type. 
You can find the explanation for each field in its documentation:
http://docs.ros.org/en/lunar/api/sensor_msgs/html/msg/NavSatFix.html

**Reply by MJy1023** [reply]
Thanks a lot!
